python-api/Makefile

```diff
@@ -21,11 +21,13 @@ lint:
 	@echo "Code style is okay!!"
 
 pep8:
-	${PEP8} gstswitch
+	# E402 module level import not at top of file
+	#  triggered by sys.path.insert(...) before gstswitch-imports
+	${PEP8} --ignore=E402 gstswitch
```

Comment:
The `--ignore=E402` should be needed as it's included in `${PEP8}` command line now. See https://github.com/timvideos/gst-switch/blob/master/python-api/Makefile#L8

This was merged in https://github.com/timvideos/gst-switch/pull/205 - probably after you had already fixed it here.